crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -418,6 +418,7 @@ impl<'a, 'o> TimelineEventHandler<'a, 'o> {
                                 reactions: Default::default(),
                                 thread_root: None,
                                 in_reply_to: None,
+                                thread_summary: None,
```

Comment:
hmm, can't stickers be part of threads? Can we add a TODO comment about this, and link it to a github issue (new or existing one), please?

Reply:
Stickers can be both part of a thread and the thread root, at least on web. The `thread_summary` is None here just because this PR doesn't populated it. I didn't consider a todo necessary as we're going to have to handle that properly soon anyway.

The `thread_root` on the other hand is a remnant of the previous (pre MsgLike) implementation where it was only available on Message types. Seeing as we're building threads properly I don't think we should bother with fixing that, I would just remove the whole thing instead.